src/ngraph/node.cpp

```diff
@@ -75,9 +125,7 @@ void Node::set_output_size(size_t n)
     NGRAPH_CHECK(n >= m_outputs.size(), "shrinking ", m_outputs.size(), " to ", n);
     for (size_t i = m_outputs.size(); i < n; ++i)
     {
-        auto tensor_descriptor = make_shared<descriptor::Tensor>(
-            element::dynamic, PartialShape::dynamic(), get_name() + "_" + to_string(i));
-        m_outputs.emplace_back(this, i, tensor_descriptor);
+        get_output_descriptor(i);
```

Comment:
Might put a comment here that this is being called for its side effect.

Reply:
`obtain_*_descriptor` ?

Reply:
`existify_descriptor`